### PR TITLE
Hide recent Registrations/Submissions when conference is over

### PR DESCRIPTION
**Pull request by selini**
Part of #462

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -33,6 +33,50 @@ def date_string(start_date, end_date)
     result
   end
 
+  # Returns time with conference timezone
+  def time_with_timezone(time)
```
Comment (differentreality) [inline app/helpers/application_helper.rb]: These functions should probably not be here

**Comment by differentreality** [comment]
@selini

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (selini) [inline app/helpers/application_helper.rb]: ok i will remove it

Comment (selini) [inline app/helpers/application_helper.rb]: done

**Comment by differentreality** [comment]
Nice work @selini :+1:

**Pull request merged**